README.md

```diff
@@ -186,17 +185,17 @@ ninja
 
 The build process generates two images:
 
-* SD-card image to be used for boot and by the Zephyr Dom0 `zephyr-dom0-xt` application;
-* USB-flash image to be used as **rootfs** by Linux operated driver domain.
+* An SD-card image, used for boot and by the Zephyr Dom0 `zephyr-dom0-xt` application;
+* A USB-flash image, used as the **rootfs** by the Linux-operated driver domain.
 
-For build system with rootfs domd on nvme storage:
+To build the system with the DomD rootfs DomD on NVMe storage, run:
```

Comment:
It seems that the second `DomD` should not be here. It should be like `with the DomD rootfs on NVMe storage`